lib/utils/helpers.js

```diff
@@ -67,21 +63,29 @@ module.exports.ChangeStartPage = function(projectRoot, plat, platformIndexUrl) {
     });
 };
 
-module.exports.ParseOptions = function(opts) {
-    var result = {};
-    opts.forEach(function(opt) {
-        var parts = opt.split(/=/);
-        var option = parts[0];
-        var value = parts[1];
+// Parses LiveReload options off of process.argv
+module.exports.ParseOptions = ParseOptions = function() {
```

Comment:
Oh I see you need ParseOptions to be visible within the module. Nit: You can just do `function ParseOptions`